Sourcery/Templates/Description.stencil

```diff
@@ -5,7 +5,8 @@
 extension {{ type.name }} {
     /// :nodoc:
     override {{ type.accessLevel }} var description: String {
-        var string = {% if type.inheritedTypes.first == "NSObject" %}"\(Swift.type(of: self)): "{% else %}super.description{% endif %}
+        var string = {% if type.inheritedTypes.first == "NSObject" %}"\(Swift.type(of: self)): "{% else %}super.description
+        string += ",\n"{% endif %}
```

Comment:
I think `\n` is not needed, just `, `

Reply:
Oh, sorry for my undestanding. I will fix it! 🚀 
Thanks 🙇‍♀️